setup.py

```diff
@@ -26,7 +26,7 @@ def get_reqs(*fns):
 
 
 setup(
-    name="django-chroniker",
```

Comment:
Is there any specific reason for this? This is a Django-specific project, not a general purpose Python package, so the convention when naming is to prefix such packages like "django-<name>".